src/components/StoreInfo/StoreImages.tsx

```diff
@@ -0,0 +1,16 @@
+export const getStoreImages = (sellerId: string | null) => {
+  return {
+    hero: `https://storage.googleapis.com/sendchinatownlove-assets/public/assets/${sellerId}/${sellerId}-hero.png`,
```

Comment:
I think we should use the URL stored in the server vs interpolating it, and we can update the server with the new URLs before we merge this